mock/config-template.js

```diff
@@ -12,6 +12,9 @@ module.exports = {
     reporterOptions: {
       currency: "chf",
       gasPrice: 21,
+      token: "ETH",
+      gasPriceApi:
+        "https://api-cn.etherscan.com/api?module=proxy&action=eth_gasPrice&apikey=YourApiKeyToken",
```

Comment:
If I may butt in on this one, it would be appreciated to note in the README what the default API URL is.

Reply:
+1. Have added a note about possibly making a table of urls and token names for easy copy-pasting and that would be a good place to put any additional info about this feature.

Reply:
Good point, also I changed the Etherscan endpoint to be compliant with [their doc](https://docs.etherscan.io/api-endpoints/geth-parity-proxy#eth_gasprice).